freud/order.py

```diff
@@ -204,60 +169,58 @@ def compute(self, orientations):
             1.0
 
         Args:
-            orientations (:math:`\left(N_{particles}, 3 \right)` :class:`numpy.ndarray`):
+            orientations \
```

Comment:
This backslash is not a line continuation because this is a raw string. It renders incorrectly in sphinx:
https://freud--1281.org.readthedocs.build/en/1281/modules/order.html#freud.order.Nematic.compute

In HOOMD-blue, I typically make the string not raw and escape all the backslashes in math blocks. This makes docstrings inconsistent (some are raw, some are not). I am not aware of a better solution to handle the long *Arg* strings.

Reply:
Noted - this is now fixed in [a205850](https://github.com/glotzerlab/freud/pull/1281/commits/a2058501fd538ea1439d3a38e1db4931d7795f13)